# HG changeset patch
# User Rik <rik@octave.org>
# Date 1427903358 25200
#      Wed Apr 01 08:49:18 2015 -0700
# Node ID 17d79913f065ad81c4e7a8fac9312564bed52bfe
# Parent  97b37077a2d373ccbc06aceb7f302d31abdd1879
Fix several typos in GUI strings (bug #44681).

* file-editor.cc (construct): "White Spaces" => "Whitespace Characters".

* settings-dialog.ui: " Normal" => "Normal".  Eliminate useless space.

* shortcut-manager.cc (do_init_data): "White Spaces" => "Whitespace Characters".

* welcome-wizard.cc (final_page): "commnity" => "community".

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -1442,17 +1442,17 @@ file_editor::construct (void)
 
   _view_editor_menu = view_menu->addMenu (tr ("&Editor"));
 
   _show_linenum_action = add_action (_view_editor_menu, QIcon (),
           tr ("Show &Line Numbers"), SLOT (show_line_numbers (bool)));
   _show_linenum_action->setCheckable (true);
 
   _show_whitespace_action = add_action (_view_editor_menu, QIcon (),
-          tr ("Show &White Spaces"), SLOT (show_white_space (bool)));
+          tr ("Show &Whitespace Characters"), SLOT (show_white_space (bool)));
   _show_whitespace_action->setCheckable (true);
 
   _show_eol_action = add_action (_view_editor_menu, QIcon (),
           tr ("Show Line &Endings"), SLOT (show_eol_chars (bool)));
   _show_eol_action->setCheckable (true);
 
   _show_indguide_action = add_action (_view_editor_menu, QIcon (),
           tr ("Show &Indentation Guides"), SLOT (show_indent_guides (bool)));
diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -123,17 +123,17 @@
                     <property name="text">
                      <string>Small</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="icon_size_normal">
                     <property name="text">
-                     <string> Normal</string>
+                     <string>Normal</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="icon_size_large">
diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -273,17 +273,17 @@ shortcut_manager::do_init_data ()
 
   init (tr ("Preferences"), "editor_edit:preferences", QKeySequence ());
   init (tr ("Styles Preferences"), "editor_edit:styles_preferences",
         QKeySequence ());
 
   // view
   init (tr ("Show Line Numbers"), "editor_view:show_line_numbers",
         QKeySequence ());
-  init (tr ("Show White Spaces"), "editor_view:show_white_spaces",
+  init (tr ("Show Whitespace Characters"), "editor_view:show_white_spaces",
         QKeySequence ());
   init (tr ("Show Line Endings"), "editor_view:show_eol_chars", QKeySequence ());
   init (tr ("Show Indentation Guides"), "editor_view:show_ind_guides",
         QKeySequence ());
   init (tr ("Show Long Line Marker"), "editor_view:show_long_line",
         QKeySequence ());
   init (tr ("Zoom In"), "editor_view:zoom_in", QKeySequence::ZoomIn);
   init (tr ("Zoom Out"), "editor_view:zoom_out", QKeySequence::ZoomOut);
diff --git a/libgui/src/welcome-wizard.cc b/libgui/src/welcome-wizard.cc
--- a/libgui/src/welcome-wizard.cc
+++ b/libgui/src/welcome-wizard.cc
@@ -206,17 +206,17 @@ final_page::final_page (welcome_wizard *
 {
   QFont ft;
   ft.setPointSize (20);
   title->setFont (ft);
 
   message->setText
   (tr ("<html><body>\n"
        "<p>We hope you find Octave to be a useful tool.</p>\n"
-       "<p>If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other commnity-based support channels.\n"
+       "<p>If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other community-based support channels.\n"
        "You can find more information about each of these by visiting <a href=\"http://octave.org/support.html\">http://octave.org/support.html</a> (opens in external browser).</p>\n"
        "</body></html>"));
   message->setWordWrap (true);
   message->setMinimumWidth (400);
   message->setOpenExternalLinks (true);
 
   QVBoxLayout *message_layout = new QVBoxLayout;
 
